src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1665,8 +1665,8 @@ static void cancel_pings(grpc_chttp2_transport* t, grpc_error* error) {
 static void send_ping_locked(grpc_chttp2_transport* t,
                              grpc_closure* on_initiate, grpc_closure* on_ack) {
   if (t->closed_with_error != GRPC_ERROR_NONE) {
-    GRPC_CLOSURE_SCHED(on_initiate, GRPC_ERROR_REF(t->closed_with_error));
-    GRPC_CLOSURE_SCHED(on_ack, GRPC_ERROR_REF(t->closed_with_error));
+    GRPC_CLOSURE_RUN(on_initiate, GRPC_ERROR_REF(t->closed_with_error));
```

Comment:
Isn't this a case where we are currently holding the combiner but are invoking closures that do not necessarily run in the combiner?  Given that, we probably want to use `GRPC_CLOSURE_SCHED()` here.

Reply:
Oh yeah, this was like a canary to check that the new combiner run still works. (I originally also thought that this would be a permanent change - I didn't take into consideration channel pings.) I'll revert this.